charts/adguard/values.yaml

```diff
@@ -17,6 +17,15 @@ primary:
     size:
     existingVolume:
 
+  route:
```

Comment:
Values should contain all possible chart values so you don't have to dig into the templates